src/web/src/app/user/model.ts

```diff
@@ -1,8 +1,10 @@
 export class User {
 	id: string;
-	name: string;
-	username: string;
+	displayName: string;
+	login: string;
 	firstname: string;
 	lastname: string;
 	email: string;
 }
+
+export type UserFields = Pick<User, 'displayName' | 'login' | 'email'>;
```

Comment:
Кажется лучше сделать отдельный тип, как ты сделал в space/model
```suggestion
export type UserFields = {
	userDisplayName: string;
	userLogin: string;
	userEmail: string;
}
```

Reply:
Готово.